### DISCUSSION TITLE
"username" repository not published at username.github.io

### DISCUSSION THREAD

**Discussion by pali-english** [community/community#57560] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello! This pertains to creating a user site per https://docs.github.com/en/pages/quickstart

A repository named the same as my username is published here:

https://pali-english.github.io/pali-english

However, I can't figure out how to get the same README.md content linked above to be published here:

https://pali-english.github.io/

The repository's Pages settings are configured to deploy from the main branch root. What do I need to do? Thanks in advance.

**Comment by airtower-luna** [comment, marked as answer]
To publish a user site your repository needs to be named like the `github.io` domain with your username, not just your username by itself (that has [another special meaning](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)). So your repository needs to be named `pali-english.github.io`. :wink:

**Reply by pali-english** [reply]
Thank you!
